mapflow/mapflow.py

```diff
@@ -2183,7 +2184,14 @@ def preview_xyz(self, provider, image_id):
                        QMessageBox.Warning)
             return
         except NotImplementedError as e:
-            self.alert(self.tr("Preview is unavailable for the provider {}").format(provider.name))
```

Comment:
Instead of just jiving an error, it alerts that OSM will be loaded instead and adds it to the bottom of Mapflow group. If OSM layer already exists, it reloads this layer: did this, so the user could see flickering in the layers panel. If it is not necessary - I'll remove this.

Reply:
Change please the icon (QMessageBox.Info) instead of Error, so that it would not look so scary